packages/sdk/src/FlagResolverClient.ts

```diff
@@ -103,13 +104,18 @@ export class FetchingFlagResolverClient implements FlagResolverClient {
     // todo refactor to move out environment
     environment,
     region,
+    customResolveBaseUrl,
   }: FlagResolverClientOptions) {
     // TODO think about both resolve and apply request logic for backends
     this.fetchImplementation = environment === 'backend' ? fetchImplementation : withRequestLogic(fetchImplementation);
     this.clientSecret = clientSecret;
     this.sdk = sdk;
     this.applyTimeout = applyTimeout;
-    this.baseUrl = region ? `https://resolver.${region}.confidence.dev/v1` : 'https://resolver.confidence.dev/v1';
+    if (environment === 'backend' && customResolveBaseUrl) {
```

Comment:
Do we need to check for environment here? I think it might be interesting to allow setting a custom url also for clients.